ci/pipelines/shared/jinja.variables.yml

```diff
@@ -149,6 +149,16 @@ tests:
   ARTIFACT_SLUG: windows-integrationtestfiles
   DUNIT_PARALLEL_FORKS: "0"
   GRADLE_TASK: integrationTest
+  GRADLE_TASK_OPTIONS: "-x geode-core:integrationTest"
+  execute_test_timeout: 6h
+  PARALLEL_DUNIT: "false"
+  PARALLEL_GRADLE: "false"
+- name: "WindowsCoreIntegration"
```

Comment:
What is the expected runtime saving by the extra job?

Reply:
great question, that's why I left this as a draft until I could measure that.  Looks like a 25% improvement (the original ~5h00m job now splits into ~1h20m (non-core) and ~3h40m (core).  Not as much savings as I had hoped, but every bit helps.  Any ideas for further splitting or pruning core tests?